queue_services/account-mailer/pyproject.toml

```diff
@@ -43,7 +45,7 @@ simple-cloudevent = { git = "https://github.com/daxiom/simple-cloudevent.py.git"
 cloud-sql-python-connector = "^1.13.0"
 
 [tool.poetry.group.dev.dependencies]
-psycopg2 = "^2.9.9"
+psycopg2 = "2.9.9"
```

Comment:
weird, thought auth was using pg8000
https://github.com/bcgov/sbc-auth/blob/8e7dbe1b653f5a8f594d8bae17e6c2b636adcf9c/auth-api/pyproject.toml#L16

Reply:
i think that was one reason there were some issues during auth-api gcp migration - unit tests were not using pg8000. will check when looking at the auth-api of things